nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -86,6 +88,19 @@ in {
         </itemizedlist>
       '';
     };
+
+    memtest86 = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = ''
+          Make MemTest86 available from the systemd-boot menu. MemTest86 is a
+          program for testing memory.  MemTest86 is a non-open-source program, so
+          this requires <literal>allowUnfree</literal> to be set to
```

Comment:
:heavy_check_mark:

Reply:
Open source != free software ;)

Reply:
^^ yheah I'm not going down that rabbit hole here and now!
My check was merely on the description clearly explaining the precondition on using this. ;)

Reply:
Yeah, I'm not looking for a discussion either :D
But the option is literally called `Unfree` - so I think the description should use the same choice words.

Reply:
In commit d88d675, I changed `non-open-source` to `unfree`.